app/views/publishers/_instant_donation_modal.html.slim

```diff
@@ -5,9 +5,13 @@
         = image_tag "icn-donation-jar@1x.png", id: 'icn-donation-jar'
         h2.modal-title = t ".headline"
         p = t ".intro"
-        = hidden_field_tag 'preferred_currency', current_publisher.wallet.default_currency
-        = hidden_field_tag 'conversion_rate', current_publisher.wallet.rates[current_publisher.wallet.default_currency]
         = link_to(t(".continue"), "#", class: 'btn btn-primary', id: "open-banner-button")
+        - if !current_publisher.wallet.default_currency.nil? && !current_publisher.wallet.default_currency.eql?('BAT')
+          = hidden_field_tag 'preferred_currency', current_publisher.wallet.default_currency
+          = hidden_field_tag 'conversion_rate', current_publisher.wallet.rates[current_publisher.wallet.default_currency]
+        - else
+          = hidden_field_tag 'preferred_currency', 'USD'
+          = hidden_field_tag 'conversion_rate', current_publisher.wallet.rates['USD']
```

Comment:
I think you need `- end`

Reply:
Is it the case for Ruby Slim?
[Slim Doc](https://www.rubydoc.info/gems/slim/frames#Control_code_-)

Reply:
Oh I suppose not.

There's also a crash on https://sentry.io/brave-software/publishers-staging-2/issues/715523160/?query=is:unresolved

Would you make it `- if current_publisher&.wallet&.default_currency.present? && !current_publisher&.wallet&.default_currency.eql?('BAT')`?